src/wallet/rpcwallet.cpp

```diff
@@ -2326,7 +2281,7 @@ static UniValue settxfee(const JSONRPCRequest& request)
     LOCK(pwallet->cs_wallet);
 
     CAmount nAmount = AmountFromValue(request.params[0]);
-    CFeeRate tx_fee_rate(nAmount, 1000);
+    CFeeRate tx_fee_rate(nAmount, 0);
```

Comment:
This will lead to a division by 0: https://github.com/VeriumReserve/verium/blob/master/src/policy/feerate.cpp#L18

Reply:
For this constructor, leaving 1000 as the second argument is probably okay. But we should have a look at other places where this constructor is used.

Reply:
I think we might need to change the code for the CFeeRate::CFeeRate(const CAmount& nFeePaid, size_t nBytes_) constructor too. But to be sure we should check where and how it is used.

Reply:
Yes, i'm looking into it :) from my point of view we should let the CFeerate generate the nSatoshiPerK it want except if it:s under 0.2 which is the minimum fee ...